airflow/hooks/dbapi.py

```diff
@@ -259,6 +266,27 @@ def get_cursor(self):
         """Returns a cursor"""
         return self.get_conn().cursor()
 
+    @staticmethod
+    def _split_sql_statements(sql, strip_comments=True, strip_semicolon=True, **kwargs):
+        """
+        Split multiple sql statements in string
+
+        :param sql: sql statements.
+        :type sql: str or list.
+        :return: splitted sql statements.
+        :rtype: list.
+        """
+        if isinstance(sql, str):
+            sql = [sql]
+
+        return [
+            s.rstrip(';') if strip_semicolon else s
```

Comment:
When would this semicolon stripping be useful?

Reply:
I believe there are certain engines, that actually will complain if `;` is passed. MsSQL particularly I believe.
I just learn couple of days that when you use CLI (via ODBC) to MsSQL, `;` at the end of SQL query does not make it execute - it will complain. Instead wheat you need to do is to type `GO` in the next line 😱

Reply:
Yes, its because what @potiuk said (this is valid for Apache Drill too)

Reply:
SQL engines are the third worst things in the world (behind browser CSS and JavaScript implementations)